main.go

```diff
@@ -469,7 +469,40 @@ func stop(c *cli.Context) error {
 }
 
 func restart(c *cli.Context) error {
-	sgs, err := getServicesOrGroups(c.Args())
+	if c.Args().First() == "all" {
```

Comment:
I'm not sure if Edward should have "special" names like this. Could you make this either a flag, or the default behavior when there are no arguments?